src/MySqlConnector/Protocol/Payloads/OkPayload.cs

```diff
@@ -1,6 +1,7 @@
+using System.Globalization;
 using System.Text;
+using MySqlConnector.Core;
 using MySqlConnector.Protocol.Serialization;
-using MySqlConnector.Utilities;
```

Comment:
This `using` is required to bring in backwards-compatibility extension methods (for older frameworks).

Reply:
changed